lib/galaxy/tool_util/verify/interactor.py

```diff
@@ -210,7 +210,6 @@ def verify_output(self, history_id, jobs, output_data, output_testdef, tool_id,
         attributes = output_testdef.attributes
         name = output_testdef.name
         expected_count = attributes.get("count")
-        self.wait_for_jobs(history_id, jobs, maxseconds)
```

Comment:
How would this have any impact on the wait time ? The first timeout would raise an exception, so you never get here ?